### ISSUE TITLE
bring_sync not writing items to shopping_list.json when added via bring! app

### ISSUE THREAD

**Issue by Flipsoo** [vlebourl/shopping_list#19]
Hey there,

i stumbled across an issue and cause i missed to sync the list correct - sorry cant delete this comment =(

Greetings

**Comment by Flipsoo** [comment]
Sorry for opening this again, i still have this weird issue so i'll try to explain whats happening and perhaps someone knows what im constantly doing wrong here:

So i select a list with `bring_select_list `and this shows me all items in the shoppinglist tab in HA frontend, items added over HA frontend and items added over the Bring App, but items added via the Bring App wont be written to the `shopping_list.json`. in this json file there are only the items added over HA frontend, using `bring_sync` doesnt solve this but i stumbled over the fact that if i check complete of one of the items added over Bring App in HA frontend the `shopping_list.json` actualizes and shows all items (HA frontend items & bring app items).

so to sum up: syncing the list with `bring_sync` shows the items added over Bring App in the HA frontend but wont write it in the `shopping_list.json` till there is some interaction with the elements in the HA frontend

sorry for bothering you again, but i cant get it solved.

Greetings

**Comment by vlebourl** [comment]
Hey guys, sorry for not answering. This repo is not under development due to lack of time... If any one is willing to claim ownership, I'll be happy to transfer it! Otherwise, I'll archive it as I won't be able to put any work into it for now...
